components/TaskCard.vue

```diff
@@ -1,24 +1,56 @@
 <template>
-  <div class="card">
-    <div class="text-description">{{ task.description }}</div>
-    <div class="text-caption margin-top-9 margin-bottom-28">
-      {{ task.createdAt }}
+  <div style="position: relative">
```

Comment:
Direct inline styling is highly discouraged. We can use an atomic class here.

Reply:
Resolved